docs/_config.yml

```diff
@@ -15,6 +15,9 @@ docs-vmr-setup: http://docs.solace.com/Solace-VMR-Set-Up/Setting-Up-VMRs.htm
 docs-core-concepts: http://docs.solace.com/Features/Core-Concepts.htm
 docs-jms-home: http://docs.solace.com/Solace-JMS-API/JMS-home-open.htm
 docs-jms-connections: http://docs.solace.com/Solace-JMS-API/Establishing-Connections.htm
+docs-jms-obtaining_connection_factories: http://docs.solace.com/Solace-JMS-API/Obtaining-Connection-Fac.htm
+docs-jms-working_with_destinations: http://docs.solace.com/Solace-JMS-API/Working-with-Destination.htm
+docs-management_tools: http://docs.solace.com/Management-Tools.htm
 docs-gm-rr: http://docs.solace.com/Solace-JMS-API/Using-Topic-Requestors.htm
```

Comment:
For these links, please follow the convention of dash separated names. I.e. replace the underscores.